src/System.Runtime/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1 @@
+MembersMustExist : Member 'System.Runtime.CompilerServices.RuntimeHelpers.GetUninitializedObject(System.Type)' does not exist in the implementation but it does exist in the contract.
```

Comment:
As we chatted we cannot add this API into System.Runtime because it cannot be supported by netstandard1.5.